Data/Sys/GameSettings/SZA.ini

```diff
@@ -6,7 +6,7 @@
 [EmuState]
 # The Emulation State. 1 is worst, 5 is best, 0 is not set.
 EmulationStateId = 4
-EmulationIssues = Config and use the quitar and drums, wiimote asks for a microphone otherwise.(r6575)
+EmulationIssues = Config and use the quitar and drums, wiimote asks for a microphone otherwise.
```

Comment:
"guitar", not "quitar", "configure", not "config" is a word, and "wiimote asks for a microphone otherwise" doesn't make sense.